syft/frameworks/torch/hook/hook_args.py

```diff
@@ -89,7 +89,7 @@
 ambiguous_functions = {"torch.unbind", "unbind", "torch.stack", "stack", "torch.mean", "torch.sum"}
 
 
-def hook_method_args(attr, method_self, args, kwargs):
+def unwrap_args_with_method(attr, method_self, args, kwargs):
```

Comment:
Actually maybe use 
hook_method_args -> unwrap_args_from_method
to improve clarity